sonata-project/admin-bundle/3.30/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Sonata\\CoreBundle\\SonataCoreBundle": ["all"],
+        "Knp\\Bundle\\MenuBundle\\KnpMenuBundle": ["all"],
```

Comment:
`Sonata\CoreBundle\SonataCoreBundle` and `Knp\Bundle\MenuBundle\KnpMenuBundle` will be added to `congig/bundles.php` automaticaly, so there is no need to describe them here.